moveit_kinematics/CMakeLists.txt

```diff
@@ -43,13 +43,14 @@ set(THIS_PACKAGE_LIBRARIES
 )
 
 set(THIS_PACKAGE_INCLUDE_DEPENDS
-  Boost
+  Eigen3
```

Comment:
Boost is included L.24. @AndyZe are you sure it's good to remove it?

Reply:
@ChrisThrasher thought it was OK. He's the expert. If not, we can make another quick PR to put it back

Reply:
I believe the thinking is, as long as these plugins build, other packages do not need to be aware of Boost. It's not part of the interface.